src/modelfree/train.py

```diff
@@ -146,14 +148,61 @@ def callback(locals, globals):
     return final_path
 
 
-@train_ex.capture
-def ppo1(batch_size, learning_rate, **kwargs):
+def _get_mpi_num_proc():
     # SOMEDAY: If we end up using MPI-based algorithms regularly, come up with a cleaner solution.
     from mpi4py import MPI
     if MPI is None:
         num_proc = 1
     else:
         num_proc = MPI.COMM_WORLD.Get_size()
+    return num_proc
+
+
+class ExpertDatasetOurFormat(ExpertDataset):
```

Comment:
Another naming nitpick, but from this classname I'd presume that the ExpertDataset itself is actually using our format, instead of what this actually seems to be doing, which is that it's reading in a format in a different style, but the dataset object itself will actually have the same format. Maybe ExpertDatasetFromOurFormat would be better, though I acknowledge that might verge on being too long for a reasonable class name

Reply:
Since it's only used in this one place I'm OK with a long class name, renamed.